src/agent/onefuzz-agent/src/main.rs

```diff
@@ -173,15 +173,15 @@ fn run(opt: RunOpt) -> Result<()> {
     }
 
     // We can't send telemetry if this fails.
-    let config = load_config(opt);
+    let rt = tokio::runtime::Runtime::new()?;
+    let config = rt.block_on(load_config(opt));
 
     if let Err(err) = &config {
```

Comment:
Maybe we can add something like the following (which we probably should've had already):
```suggestion
    // We can't send telemetry, because we couldn't get a telemetry key from the config.
    // Instead, log to an assumed-redirected stdout for the sake of debugging.
    if let Err(err) = &config {
```